atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/ThriftObjectSizeUtils.java

```diff
@@ -64,9 +67,9 @@ public static Map<String, Long> getSizeOfMutationPerTable(
             long keySize = ThriftObjectSizeUtils.getByteBufferSize(key);
 
             tableToMutations.forEach((table, mutations) -> {
-                Long size = tableToSize.getOrDefault(table, 0L);
```

Comment:
yeah this is one of those where I wonder what the original writers were thinking, though it probably made sense at the time